ycsb-mongodb/mongodb/src/main/java/com/yahoo/ycsb/db/MongoDbClient.java

```diff
@@ -48,6 +49,9 @@
 import java.util.UUID;
 import java.util.Vector;
 import java.util.concurrent.atomic.AtomicInteger;
+import org.bson.*;
```

Comment:
nits: can you have your IDE optimize imports?

Reply:
done